test/com/facebook/buck/jvm/kotlin/KotlinBuckConfigTest.java

```diff
@@ -17,168 +17,190 @@
 
 import static java.io.File.pathSeparator;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 
 import com.facebook.buck.cli.BuckConfig;
 import com.facebook.buck.cli.FakeBuckConfig;
 import com.facebook.buck.io.MoreFiles;
 import com.facebook.buck.io.ProjectFilesystem;
-import com.facebook.buck.rules.PathSourcePath;
+import com.facebook.buck.log.Logger;
 import com.facebook.buck.testutil.integration.ProjectWorkspace;
 import com.facebook.buck.testutil.integration.TemporaryPaths;
 import com.facebook.buck.testutil.integration.TestDataHelper;
 import com.facebook.buck.util.HumanReadableException;
 import com.google.common.collect.ImmutableMap;
-import java.io.IOException;
-import java.nio.file.Path;
+
 import org.hamcrest.Matchers;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 
+import java.io.IOException;
+import java.nio.file.Path;
+
 public class KotlinBuckConfigTest {
 
   @Rule public TemporaryPaths tmp = new TemporaryPaths();
 
   private ProjectWorkspace workspace;
+  private Path testDataDirectory;
 
   @Before
   public void setUp() throws InterruptedException, IOException {
     KotlinTestAssumptions.assumeUnixLike();
-
-    workspace = TestDataHelper.createProjectWorkspaceForScenario(this, "kotlin_compiler_test", tmp);
-    workspace.setUp();
+    workspace = TestDataHelper
```

Comment:
I don't think we should create a new testdata directory for this buck config test.  That's pretty heavy when you can just use tmp to create temporary files/directory structures.

Check out how `DBuckConfigTest`,  does it.  We should do it that way and delete the `testdata/faux_kotlin_home` and `testdata/kotlin_compiler_test` directories.